CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# Contributing to Prettier
+# Contributing to Prettierx
```

Comment:
Changes like this would be more consistent with README.md. I would be happy to take care of this.

```suggestion
# Contributing to prettierX
```

Reply:
Go for it!